ebusd/config.yaml

```diff
@@ -33,7 +33,7 @@ options:
   mqttint: "/etc/ebusd/mqtt-hassio.cfg"
   mqttjson: true
 schema:
-  mode: "list(enh|ens|udp)?"
+  mode: "list(enh|ens|udp)"
```

Comment:
Any reason the mode is enforced now?

Reply:
@tjorim at least in my case, I did not select the mode, and the addon was not working without it. I am assuming the mode is necessary to be selected, otherwise the adapter is not working, so I thought it would be wise to make it mandatory - if there is a scenario, where this does not make sense, let me know and I can change it back.